htdocs/compta/facture/card.php

```diff
@@ -2742,7 +2742,7 @@
 		// Invoice situation
 		if (getDolGlobalInt('INVOICE_USE_SITUATION') == 2) {
 			$previousprogress = $line->get_allprev_progress($line->fk_facture);
-			$fullprogress = price2num(GETPOST('progress', 'alpha'));
+			$fullprogress = price2num(GETPOST('progress', 'alpha'), 'MU');
```

Comment:
The 'MU' is related to amount, so varies with setup and may be 0.
For consistent result, i recommend for percentage to use 2 instead of 'MU'

Reply:
Done